Polygon.js

```diff
@@ -0,0 +1,85 @@
+{
+	"translatorID": "fa7c37b1-fda4-418a-a8b8-2491929411ab",
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?://(?:www\\.)polygon\\.com",
+	"minVersion": "1.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2015-12-05 20:54:17"
+}
+
+// http://en.wikipedia.org/wiki/User_talk:Czar
+
+function detectWeb(doc, url) {
+	if (doc.title.match("| Polygon")) {
+ 		return "webpage";
+ 	}
+	
+	return null;
+}
+
+function scrape(doc, url) {
+	var namespace = doc.documentElement.namespaceURI; 
+	var nsResolver = namespace ? function(prefix) {
+		if (prefix == 'x') return namespace; else return null; 
+		} : null; 
+	
+	var newItem = new Zotero.Item("webpage");
+	newItem.url = doc.location.href;
+	newItem.language = "en";
+	newItem.publicationTitle="Polygon";
+	
+		// Article title
+	var soughtXPath = "//h2";
+		evalXPath = doc.evaluate(soughtXPath, doc, nsResolver, XPathResult.ANY_TYPE, null).iterateNext().textContent;
+	newItem.title = evalXPath;
+	
+		// Authors
+	var authorCounter = 0;
+	var authorArray = new Array();
```

Comment:
Issue with this is multiple authors, e.g., [Nintendo brings Mother ...](http://www.polygon.com/2015/6/14/8779193/earthbound-mother-famicom-wii-u-virtual-console-nintendo):
`<meta name="author" content="Michael McWhertor">`
Unless I pull from the `<!-- GTM DataLayer -->` section? Thoughts? This is an edge case but I'd like to be able to handle it